e2e/apprunner/apprunner_suite_test.go

```diff
@@ -17,6 +17,8 @@ import (
 var cli *client.CLI
 var appName string
 
+const ssmName = "e2e-apprunner-ssm-param-secret"
```

Comment:
Do we need to include time info to make them unique 🤔

Reply:
I went with the route where we "not error out when the secret already exists" instead of "creating unique secrets". Seems unnecessary to use a new secret every time.